File: app/models/ingredient.rb
# == Schema Information
#
# Table name: ingredients
#
#  id          :integer          not null, primary key
#  name        :text
#  product     :text
#  qty         :text
#  cocktail_id :integer
#  created_at  :datetime         not null
#  updated_at  :datetime         not null
#
# Indexes
#
#  index_ingredients_on_cocktail_id  (cocktail_id)
#

class Ingredient < ActiveRecord::Base
  belongs_to :cocktail
end

File: app/client/cocktails/Ingredient.js
import React from 'react';
import _ from 'lodash';
import { connect } from 'react-redux';

const Ingredient = ({ name, qty, product }) => {
  return (
    <li>
      {qty} {!_.isEmpty(product) ? <a href={product && `${product}`}>{name}</a> : name }
    </li>
  );
};

export default connect()(Ingredient);
